backend/services/questionService.js

```diff
@@ -0,0 +1,19 @@
+const { Question } = require("../models/question");
+
+const getAll = async () => {
+  const questions = await Question.find().populate("mentor");
```

Comment:
Use `populate` method to populate the mentor data automatically.

https://mongoosejs.com/docs/populate.html